decidim-accountability/spec/forms/admin/result_form_spec.rb

```diff
@@ -87,28 +87,31 @@ module Decidim::Accountability
     end
 
     context "with proposals" do
+      subject { described_class.from_model(result).with_context(context) }
+
       let(:proposals_feature) { create :feature, manifest_name: :proposals, participatory_space: participatory_process }
       let!(:proposal) { create :proposal, feature: proposals_feature }
 
+      let(:result) do
+        create(
+          :result,
+          feature: current_feature,
+          scope: scope,
+          category: category
+        )
+      end
+
       describe "#proposals" do
+        before do
+          result.link_resources([proposal], "included_proposals")
+        end
```

Comment:
Can you add a line between the `before` block and the `it` block to improve readability, please?